.github/workflows/windows-periodic.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         with:
           azcliversion: latest
           inlinescript: |
-            az group create -n ${{ matrix.AZURE_RESOURCE_GROUP }} -l ${{ env.AZURE_DEFAULT_LOCATION }}
+            az group create -n ${{ matrix.AZURE_RESOURCE_GROUP }} -l ${{ env.AZURE_DEFAULT_LOCATION }} --tags creationTimestamp=$(date +%Y-%m-%dT%T%z)
```

Comment:
I'm curious.. would `date --iso-8601=seconds` also work?

Reply:
I'll give it a try to make sure all the tools involved play nice and come back to you.

Reply:
> I'll give it a try to make sure all the tools involved play nice and come back to you.

thx!

Reply:
> > I'll give it a try to make sure all the tools involved play nice and come back to you.
> 
> thx!

Yeah, so this particular command is executed from a container running: azure-cli:latest . Doesn't seem date inside there knows about `--iso-8601` :(

```
Starting script execution via docker image mcr.microsoft.com/azure-cli:latest
date: unrecognized option: iso-8601=seconds
BusyBox v1.32.1 () multi-call binary.

Usage: date [OPTIONS] [+FMT] [TIME]

Display time (using +FMT), or set time

	[-s,--set] TIME	Set time to TIME
	-u,--utc	Work in UTC (don't convert to local time)
	-R,--rfc-2822	Output RFC-2822 compliant date string
	-I[SPEC]	Output ISO-8601 compliant date string
			SPEC='date' (default) for date only,
			'hours', 'minutes', or 'seconds' for date and
			time to the indicated precision
	-r,--reference FILE	Display last modification time of FILE
	-d,--date TIME	Display TIME, not 'now'
	-D FMT		Use FMT (strptime format) for -d TIME conversion

Recognized TIME formats:
	hh:mm[:ss]
	[YYYY.]MM.DD-hh:mm[:ss]
	YYYY-MM-DD hh:mm[:ss]
	[[[[[YY]YY]MM]DD]hh]mm[.ss]
	'date TIME' form accepts MMDDhhmm[[YY]YY][.ss] instead
***
```

Reply:
thx for trying!